docs/setup.md

```diff
@@ -78,6 +78,7 @@ You have to configure the application to interact with your teams.
    - Windows: Press `Alt` key to bring up the menu at the top of the window, then click `File -> Settings`.
    - OS X: Click `Mattermost` from the menu at the top of the screen, then click `Preferences...`.
    - Linux: Click `File -> Settings` on the menu.
+   - If tray menu : Right-click on tray icon and click `Settings`
```

Comment:
It's an option in OS X and Linux, but I feel we don't have to mention about it here. If the user don't use the tray icon, the sentence would be ignored.